src/fidesops/main.py

```diff
@@ -173,7 +176,9 @@ def generate_route_file_map() -> None:
 
                 rule = re.compile(r"^" + path)
 
-                route_file_map[rule] = FileResponse(str(filepath.relative_to(".")))
+                route_file_map[rule] = FileResponse(
+                    f"/admin_ui/{str(filepath.relative_to(WEBAPP_DIRECTORY))}"
```

Comment:
This will help prevent errors if the directory name changes in the future

```suggestion
                    f"{WEBAPP_DIRECTORY}/{filepath.relative_to(WEBAPP_DIRECTORY)}"
```